File: HardwareInfoCollector/models/aio.py
from models.hardware import *

class AIO(Hardware):
    FansCount: int

    def __init__(self, product_type: int, model: str, price: int, link: str, fans_count: int):
        super().__init__(product_type, model, price, link)
        self.FansCount = fans_count

    def __str__(self):
        return f'{self.ProductType} {self.Model} {self.Price} {self.Link} {self.FansCount}'

File: BuildGenerator/src/Bll/Models/Hardware/Aio.cs
using Bll.Models.Hardware.Abstractions;

namespace Bll.Models.Hardware;

public class Aio : Cooling
{
    public sealed override double Price { get; set; }
    public sealed override string Link { get; set; } = string.Empty;
    public sealed override string Model { get; set; } = string.Empty;
    
    public int FansCount { get; set; }

    public Aio()
    {
        
    }

    public Aio(double price, string link, string model, int fansCount)
    {
        Price = price;
        Link = link;
        Model = model;
        FansCount = fansCount;
    }
}